topics/design-system/index.md

```diff
@@ -0,0 +1,8 @@
+---
+display_name: Design system
+short_description: A design system is a way to have modular and reusable CSS components as well as a separation of concerns.
```

Comment:
I feel like this description of design systems is too narrow. Design systems are not only for CSS components but rather for any well designed, modular, reusable software components.

Reply:
But isn't it primarily CSS because at its core it's about formalizing a style guide to be reusable and stay consistent with corporate identity. Some bigger design systems step up their game with JavaScript [web components](https://github.com/topics/web-components).